src/css/profile/mobile/common/button.less

```diff
@@ -567,6 +567,15 @@ a.ui-btn {
 	}
 }
 
+@media all and (min-width: 673px) and (min-height: 411px) {
+	.ui-btn {
+		&.ui-btn-contained:not(.ui-btn-inline) {
+			max-width: 60%;
```

Comment:
does this also determine default width to 60%?

Reply:
Default width for `contained` button is defined in line 505. I think, default width is common for different resolutions.

Reply:
ok